cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -28,9 +28,11 @@ After you create the pipelines for your application, you can view and visually i
 
 include::modules/op-constructing-pipelines-using-pipeline-builder.adoc[leveloffset=+1]
 
-== Creating applications with OpenShift Pipelines
+== Creating OpenShift Pipelines along with applications
+
+To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You see a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. You can view all of your available pipelines and select the pipelines you want to use to create applications while importing your code or deploying an image. If an administrator has opted in for the Tekton Hub Integration, you can see tasks from Tekton Hub that are supported by your cluster. You can also check whether a webhook URL exists for a generated pipeline.
```

Comment:
@karthikjeeyar can you please confirm if the last three sentences are technically correct?